compiler/main.bal

```diff
@@ -25,18 +31,31 @@ public function main(string[] filenames, *Options opts) returns error? {
             check showTypes(filename);
             continue;
         }
-        check compileFile(filename, check chooseOutputFilename(filename, opts.outDir));
+        check compileFile(filename, check chooseOutputFilename(filename, opts.outDir), { gcName: gc });
     }  
 }
 
+function validateGC(string? gcName) returns string|error? {
```

Comment:
I suggest `validGcName` as the name of this function. I prefer to think of it declaratively rather than procedurally: it is a function that returns a valid GC name or fails.  I also think `check validGcName(x)` reads nicely.

Reply:
Fixed